### PR TITLE
feat(ranger): add action for extension-request

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR helps automate how we handle extension request issues. It also updates the docs for installing OpenVSX extensions via the command line.

TODOS
- [x] refactor docs based on feedback

### CODE REVIEW

File: .github/ranger.yml [hunk 1]
```diff
@@ -23,3 +25,7 @@ labels:
     action: comment
     delay: 5s
     message: "Thanks for making your first contribution! :slightly_smiling_face:"
+  extension-request:
+    action: close-faster
+    delay: 5s
+    message: "Thanks for opening an extension request! We are currently in the process of switching extension marketplaces and transitioning over to [Open VSX](https://open-vsx.org/). Once https://github.com/eclipse/openvsx/issues/249 is implemented, we can fully make this transition. Therefore, we are no longer accepting new requests for extension requests. We suggest installing the VSIX file and then installing into code-server as a temporary workaround. See [docs](https://github.com/cdr/code-server/blob/main/docs/FAQ.md#installing-vsix-extensions-via-the-command-line) for more info."
```
Comment (jsjoeio) [inline .github/ranger.yml:31]: Is there a way to wrap this? And does the link look right?

Comment (jawnsy) [reply .github/ranger.yml:31]: Yes! You can do long lines in YAML using `|` or `>` https://en.wikipedia.org/wiki/YAML#Basic_components

Try:

```yaml
extension-request:
  message: >
    Thanks for opening an extension request!
    We are currently in the process of
```

This is if you want it to be a single paragraph, you can also use `|` to preserve the newlines if you want (which of course might not make a difference if this message is Markdown)

Comment (jsjoeio) [reply .github/ranger.yml:31]: TIL!
You're the man! Thanks @jawnsy

File: docs/FAQ.md [hunk 2]
```diff
@@ -132,6 +133,21 @@ code-server --install-extension ms-python.python
 code-server --install-extension downloaded-ms-python.python.vsix
 ```
 
+### Installing VSIX Extensions via the Command Line
```
Comment (code-asher) [inline docs/FAQ.md:136]: I think this overlaps a little with the previous paragraph that lists how to install extensions on the CLI.

What if we move that into this heading, then create another heading for this content with something like "Installing OpenVSX extensions via the command line" for this content?

We could also mention they can switch early to OpenVSX by setting `SERVICE_URL` and `ITEM_URL` (we already have a section so maybe we just link it from here).

Comment (code-asher) [reply docs/FAQ.md:136]: Actually maybe we should put this under "How can I request a missing extension?"

Comment (jsjoeio) [reply docs/FAQ.md:136]: Agreed.

Yes! Let's do that!

Comment (code-asher) [reply docs/FAQ.md:136]: Or just remove that section.

Comment (jsjoeio) [reply docs/FAQ.md]: I opted to add it under "How can I request a missing extension" - I say we move remove the extension request section once we fully switch to OpenVSX

Comment (code-asher) [reply docs/FAQ.md]: My thinking is that the flow from a user might look like:

"How do I install an extension?" 
"Oh, it's not available, how do I request it?"
"Ah, they aren't accepting requests, how do I install it then?"
"OK, I can either do it manually or download from Open VSX."

Comment (jsjoeio) [reply docs/FAQ.md]: Ahh yeah good point about that. Okay I'll make these changes then re-request a review! Thanks for adding the structure/flow too! That helps a ton!!

Comment (jsjoeio) [reply docs/FAQ.md]: Embarrassingly, I didn't know you could do it this way, hence the `curl` suggestion:
```
SERVICE_URL=https://open-vsx.org/vscode/gallery ITEM_URL=https://open-vsx.org/vscode/item code-server --install-extension stylelint.vscode-stylelint
```

File: docs/FAQ.md [hunk 3]
```diff
@@ -82,15 +84,43 @@ These are the closed source extensions presently unavailable:
...
-See below for installing an extension from the cli.
+If an extension is not available from the marketplace or does not work, you can
+grab its VSIX from its GitHub releases or build it yourself.
+
+Once you have downloaded the VSIX to the remote machine you can either:
+
+- Run the `Extensions: Install from VSIX` command in the Command Palette.
+- Use `code-server --install-extension <path to vsix>`
+
+If the extension exists on another marketplace, you can also download it from there using the same command. For instance, downloading off OpenVSX can be done like this:
```
Comment (jsjoeio) [inline docs/FAQ.md:119]:
```suggestion
You can also download extensions from the command line. For instance, downloading off OpenVSX can be done like this:
```

**Comment by jsjoeio** [comment]
bump @code-asher ready for another review

**Review by code-asher — APPROVED** [review]

**Pull request merged**